mlflow/R/mlflow/R/tracking-experiments.R

```diff
@@ -46,12 +46,36 @@ mlflow_list_experiments <- function(view_type = c("ACTIVE_ONLY", "DELETED_ONLY",
     tibble::as_tibble()
 }
 
+#' Set Experiment Tag
+#'
+#' Sets a tag on an experiment. Tags are experiment metadata that can be updated.
+#'
+#' @param key Name of the tag. Maximum size is 255 bytes. This field is required.
+#' @param value String value of the tag being logged. Maximum size is 500 bytes. This field is required.
```

Comment:
Let's update this to reflect limit changes. We can use the same string as the one in the proto.